android/src/main/java/com/theweflex/react/WeChatModule.java

```diff
@@ -441,6 +471,10 @@ public void onResp(BaseResp baseResp) {
         } else if (baseResp instanceof SendMessageToWX.Resp) {
             SendMessageToWX.Resp resp = (SendMessageToWX.Resp) (baseResp);
             map.putString("type", "SendMessageToWX.Resp");
+        } else if (baseResp instanceof PayResp) {
+            PayResp resp = (PayResp) (baseResp);
+            map.putString("type", "PayReq.Resp");
+            map.putString("returnkey", resp.returnKey);
```

Comment:
Have corrected? Did you push your code in live? I don't see any changes on this PR :(